### DISCUSSION TITLE
v9.4.2 does not work on iOS 10

### DISCUSSION THREAD

**Discussion by Bunlong** [vercel/next.js#13421] [Help]
# Bug report

## Describe the bug

After I upgraded Next.js version 9.3.6 to 9.4.2 I got the error on iOS 10 as following:

![100997471_686480498840915_2649637110985785344_n](https://user-images.githubusercontent.com/1308397/82973879-c3963000-a002-11ea-933c-1f5d46cdc6f7.jpg)

## System information

- OS: Ubuntu
- Browser chrome, safari on iOS 10
- Version of Next.js: 9.4.2
- Version of Node.js: 12.16.1

**Comment by jamesmosier** [comment]
I wasn't able to reproduce this on an iPhone 7 running iOS 10.

Do you have a repository I could test with? @Bunlong
